build/buildconstants/params_interop.go

```diff
@@ -8,16 +8,13 @@ import (
 	"strconv"
 
 	"github.com/ipfs/go-cid"
-	logging "github.com/ipfs/go-log/v2"
 
 	"github.com/filecoin-project/go-address"
 	"github.com/filecoin-project/go-state-types/abi"
 	"github.com/filecoin-project/go-state-types/network"
 	builtin2 "github.com/filecoin-project/specs-actors/v2/actors/builtin"
 )
 
-var log = logging.Logger("buildconstants")
```

Comment:
What about the log call on Line 86, what logger does it call?

Reply:
there's one in build/buildconstants/shared_funcs.go that they all should use, however it should probably be changed from `"build/buildtypes"` to `"build/buildconstants"` @snadrus